README.rst

```diff
@@ -50,55 +50,89 @@ Currently available datasets
-long term.
+In order to be able to share large amounts of public data without being exposed to large
+unexpected bills from Google due to someone maliciously or accidentally downloading a
+large volume of data programmatically, we've set the cloud storage to use `requester
+pays <https://cloud.google.com/storage/docs/requester-pays>`__. This means the person
+downloading the data is responsible for those (modest) costs instead. Downloading all of
+the EPA CEMS, FERC 1, PUDL, and US Census data we're publishing from North America will
+cost around $0.75, and it will be cached locally so that it's not downloaded again until
+a new version is released.
 
-The idea is that for any released version of PUDL, you should also be able to
-install a corresponding data catalog, and know that the software and the data
-are compatible. You can also install just the data catalog with minimal
-dependencies, and not need to worry about the PUDL software that produced it at
-all, if you simply want to access the DBs or Parquet files directly.
+To set up a GCP billing project and use it for authentication when accessing the
+catalog:
 
-In development, this arrangement will mean that every morning you should have
-access to a fully processed set of data products that reflect the branch of code
-that you’re working on, rather than the data and code getting progressively
-further out of sync as you do development, until you take the time to re-run the
-full ETL locally yourself.
+* `Create a project on GCP <https://cloud.google.com/resource-manager/docs/creating-managing-projects#creating_a_project>`__;
+  if this is the first time using GCP, a prompt should appear asking you to choose which
+  Google account to use for your GCP-related activities. (You should also receive $300
+  in initial cloud credits.
+* `Create a Cloud Billing account <https://cloud.google.com/billing/docs/how-to/manage-billing-account#create_a_new_billing_account>`__
+  associated with the project and `enable billing for the project
+  <https://cloud.google.com/billing/docs/how-to/modify-project#enable_billing_for_a_project>`__
+  through this account.
+* `Using Google Cloud IAM <https://cloud.google.com/iam/docs/granting-changing-revoking-access#granting-console>`__,
+  add the **Service Usage Consumer** role to your account, which enables it to make
+  billed requests on the behalf of the project.
+* Install the `gcloud utilities <https://cloud.google.com/sdk/docs/install>`__ on your
+  computer. This can be done using ``conda`` (or ``mamba``):
+
+.. code:: bash
+
+  conda install -c conda-forge google-cloud-sdk
+
+* Initialize the ``gcloud`` command line interface, logging into the account used to
+  create the aforementioned project and selecting it as the default project; this will
+  allow the project to be used for requester pays access through the command line:
+
+.. code:: bash
+
+  gcloud auth login
+  gcloud init
```

Comment:
this step asked me some questions:

```
Pick configuration to use:
 [1] Re-initialize this configuration [default] with new settings 
 [2] Create a new configuration
```

Reply:
it also asked:
```
Pick cloud project to use: 
 [1] northern-hope-350719
 [2] pudl-test
 [3] Enter a project ID
 [4] Create a new project
```

(I had just made the `pudl-test` project when i first logged in)

Reply:
i choose 2

Reply:
Which one did you choose? I think it should be the project you just created, and within which you should have given yourself the permission to spend money.